test/parallel/test-fs-watchfile.js

```diff
@@ -37,24 +36,13 @@ const expectedStatObject = new fs.Stats(
     Date.UTC(1970, 0, 1, 0, 0, 0)             // birthtime
 );
 
-function removeTestFile() {
-  try {
-    fs.unlinkSync(enoentFile);
-  } catch (ex) {
-    if (ex.code !== 'ENOENT') {
-      throw ex;
-    }
-  }
-}
-
-// Make sure that the file does not exist, when the test starts
-removeTestFile();
+common.refreshTmpDir();
 
 // If the file initially didn't exist, and gets created at a later point of
 // time, the callback should be invoked again with proper values in stat object
 var fileExists = false;
 
-fs.watchFile(enoentFile, common.mustCall(function(curr, prev) {
+fs.watchFile(enoentFile, {interval: 0}, common.mustCall(function(curr, prev) {
```

Comment:
I figured that would poll every time the event loop finished. Ultimately, `interval` gets passed to libuv's `uv_fs_poll_start()`. Judging from [this line of code](https://github.com/nodejs/node/blob/aa33db32380fe05a3bc7a2e320bbf63a6adddf08/deps/uv/src/fs-poll.c#L77) it seems that `0` gets changed to `1`.